src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+const inputsArr = [...inputs];
```

Comment:
```suggestion
const inputs = [...document.querySelectorAll('input')];
```
arr, array, obj, object - всегда плохая идея